messagequeue/chunkeroptions.go

```diff
@@ -0,0 +1,50 @@
+package messagequeue
+
+import (
+	"time"
+
+	"github.com/beaconsoftwarellc/gadget/v2/errors"
+)
+
+const (
+	defaultChunkSize                  = 10
+	defaultEntryMaxWait time.Duration = 10 * time.Second
```

Comment:
nit: either make this `defaultMaxElementWait` or change `MaxElementWait` to `EntryMaxWait`

Reply:
also in messagequeue/enqueueroptions.go, you follow a pattern of min/max/default.  I like the consistency of having the different value type clustered and in a consistent sequence.  You should follow that same pattern here.

Reply:
good call :+1: